bigtable/client/mutations.h

```diff
@@ -62,29 +62,138 @@ constexpr std::int64_t ServerSetTimestamp() { return -1; }
  * @name Create mutations to delete a range of cells from a column.
  *
  * The following functions create a mutation that deletes all the
- * cells in the given column family and column within the given
- * timestamp range.
+ * cells in the given column family and, column within the given
+ * timestamp in the range.
+ *
+ * The function accepts any instantiation of `std::chrono::duration<>` for the
+ * @p timestamp_begin and @p timestamp_end parameters.  For example:
+ *
+ * @code
+ * using namespace std::chrono_literals; // C++14
+ * bigtable::DeleteFromColumn("fam", "col", 0us, 10us)
+ * @endcode
  *
  * The ending timestamp is exclusive, while the beginning timestamp is
  * inclusive.  That is, the interval is [@p timestamp_begin, @p timestamp_end).
- * The value 0 is special and treated as "unbounded" for both the begin and end
- * endpoints of the time range.  The Cloud Bigtable server rejects invalid and
- * empty ranges, i.e., any range where the endpoint is smaller or equal than to
- * the initial endpoint unless either endpoint is 0.
+ * The value 0 is special and treated as "unbounded" for both the begin and
+ * end endpoints of the time range.  The Cloud Bigtable server rejects
+ * invalid and empty ranges, i.e., any range where the endpoint is smaller or
+ * equal than to the initial endpoint unless either endpoint is 0.
+ *
+ * @tparam Rep1 a placeholder to match the Rep tparam for @p timestamp_begin
+ * type. The semantics of this template parameter are documented in
+ * std::chrono::duration<>` (in brief, the underlying arithmetic type
+ * used to store the number of ticks), for our purposes it is simply a
+ * formal parameter.
+ *
+ * @tparam Rep2 similar formal parameter for the type of @p timestamp_end.
+ *
+ * @tparam Period1 a placeholder to match the Period tparam for
+ * @p timestamp_begin type. the semantics of this template parameter are
```

Comment:
s/type. the/type. The/

Reply:
If I am correct then,  "type. the semantics" should be "type. The semantics" right ?

Reply:
Yes.